### ISSUE TITLE
Use first assembly for ribbon in 3D view

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#1373]
Ribbon diagram should be generated for the chains that define the first assembly in `assemblies.yaml`.  

Presumably B/E that must tell F/E what that is.  

(We haven't discussed that ribbon for years, but now is the time to fix it.  Could be yellow release if tricky.)

(This will look completely right once #1369 is done.)

**Comment by phraenquex** [comment]
Almost certainly related to the answer to #1369

**Comment by mwinokan** [comment]
The ribbon shown in NGL needs to be taken from the first assembly in the [assemblies.yaml](https://github.com/xchem/xchem-align/blob/master/USER-GUIDE.md#22-the-assemblies-yaml) of the alignment. I.e taking chains from the reference PDB file.

This might not look right until #1369 is resolved if the ligand displayed is from a different assembly

@phraenquex to think about how the frontend processes the reference PDBs (and potentially where they should be stored in the XCA output)

**Comment by mwinokan** [comment]
waiting on #1292

**Comment by mwinokan** [comment]
@kaliif @boriskovar-m2ms will need an endpoint to query where the reference assembly can be found for the ribbon to be rendered. 

Currently the reference structures are not loaded

**Comment by mwinokan** [comment]
@kaliif if this is a quick fix, please work on this before #1392

**Comment by mwinokan** [comment]
@kaliif has completed the backend part.

@boriskovar-m2ms has pulled the latest backend, and now does not have the latest compatible data. @tdudgeon will provide Boris with A71EV2A data and @mwinokan with an RHS upload

**Comment by boriskovar-m2ms** [comment]
@mwinokan do we need an actual endpoint because in the target there is a ```template_protein``` field from where the ```.pdb``` to render ribbon is taken.

**Comment by kaliif** [comment]
If this field is not used for anything else, I can point it to the correct pdb file (atm it's just the pdb from the first available site observation).

**Comment by boriskovar-m2ms** [comment]
Just checked the code and it's not used for anything else.

**Comment by mwinokan** [comment]
@kaliif has moved his b/e changes to staging. @boriskovar-m2ms will move his f/e changes to staging tomorrow

**Comment by mwinokan** [comment]
@boriskovar-m2ms is your latest frontend code in staging? The ribbon does not look correct for [A71EV2A](https://fragalysis.xchem.diamond.ac.uk/viewer/react/projects/42/44)

**Comment by boriskovar-m2ms** [comment]
> @boriskovar-m2ms is your latest frontend code in staging? The ribbon does not look correct for [A71EV2A](https://fragalysis.xchem.diamond.ac.uk/viewer/react/projects/42/44)

yes it is but most worryingly the snapshot doesn't work for me. I need to investigate both issues.

**Comment by boriskovar-m2ms** [comment]
OK false alarm. Snapshot doesn't work for me because I can't see A71EV2A target. Maybe if target is not found (because it was deleted or we can not see it - there is no way for frontend to tell the difference) we should notify user so users are not staring at pretty much empty screen.

**Comment by mwinokan** [comment]
Sorry I think the snapshot is my fault. I deleted the compound set after making the snapshot (still something to fix I suppose)
